boilerplate/app/services/reactotron/reactotron.ts

```diff
@@ -1,203 +1,194 @@
-import { Tron } from "./tron"
```

Comment:
It looks like this file has a ton of changes, but primarily it was just de-"environment"-alizing it and removing the class structure. Instead, it's just some local variables and exported functions. Much simpler.